docs/tools/cli/service/integration.md

```diff
@@ -86,6 +86,10 @@ avn service integration-endpoint-create --endpoint-name demo-ext-kafka \
     --user-config-json  '{"bootstrap_servers":"servertest:123","security_protocol":"PLAINTEXT"}'
 ```
 
+:::note
+Further examples of external Apache Kafka® endpoint creations are documented [here](/docs/products/flink/howto/ext-kafka-flink-integration#step-4-create-an-external-apache-kafka-endpoint).
```

Comment:
```suggestion
For more examples of creating external Apache Kafka® endpoints, see [Integrate Aiven for Apache Flink® with Apache Kafka®](https://aiven.io/docs/products/flink/howto/ext-kafka-flink-integration#step-4-create-an-external-apache-kafka-endpoint).
```

To improve SEO, consider avoiding [here] for links and use the topic name or similar instead.

Reply:
ok. should the link not start with /docs/?

Reply:
Sorry, that part is correct, should start with /docs/

Reply:
done